common/network-common/src/main/java/org/apache/spark/network/client/TransportClientFactory.java

```diff
@@ -210,6 +210,18 @@ private TransportClient createClient(InetSocketAddress address)
       .option(ChannelOption.CONNECT_TIMEOUT_MILLIS, conf.connectionTimeoutMs())
       .option(ChannelOption.ALLOCATOR, pooledAllocator);
 
+    if (conf.backLog() > 0) {
+      bootstrap.option(ChannelOption.SO_BACKLOG, conf.backLog());
```

Comment:
It is really used for Client bootstrap? AFAIK backlog is mainly used in server side to cache the incoming connections.